ethereum/chain.py

```diff
@@ -263,21 +264,21 @@ def add_block(self, block):
                         break
                     b = self.get_parent(b)
                 replace_from = b.header.number
-                for i in xrange(replace_from, 2**63 - 1):
+                for i in islice(count(), replace_from, 2**63 - 1):
```

Comment:
The API for `count` suggests that you can do `count(replace_from)` and avoid the use of `islice` here.